docs/_customer-interactive-api/_example-Leaderboard.md

```diff
@@ -1,17 +1,43 @@
 ```json
 {
-  "leaderboardPageId": "1212",
-  "broadcastId": "__8example*asd9",
-  "version": 2,
-  "entries": {
-    "score": 1000,
-    "rank": 1,
-    "displayName": "example",
-    "avatarUrl": "",
-    "userId": "example",
-    "attributes": {}
-  },
-  "pageNumber": 1,
-  "maxPage": 10
+  "leaderboardPageId": "____591DjYVBDjB6CNYAAA==",
```

Comment:
Some of the fields (broadcastId, etc) of LeaderboardPageDTO seem to be removed now.
Is this intentional?